app/models/user.rb

```diff
@@ -88,6 +88,10 @@ def is_test?
     !!is_test
   end
 
+  def using_openstax?
+    false
+  end
+
```

Comment:
Defining this is not necessary because, I believe, Rails dynamically defines it the moment you add the database column `using_openstax` to the users table. It actually might be problematic if you define it here as it will always be false regardless of the actual value in the database for `using_openstax`.

Reply:
@BryanEliDimas I'm removing it, but just for my understanding, why do other boolean values in that model have the function defined, but this one should not?

Reply:
None of the methods correspond to a database field by the same name without the `?` so we must define them, except for `is_test`, in which case I don't know the reason why 🤔. It looks like the intent was to cast it into a boolean (because of the double bang `!!`) but it's already a boolean in the database so in that case, defining that method is unnecessary but **not** problematic because it gets the value from the DB instead of a hardcoded `true` or `false`.

Reply:
We could (and should attempt to) remove `def is_test?`, and the tests should still pass. If they don't, then I must be wrong **or** it doesn't work because of some corner case with a gem.

Reply:
Removed